setup.py

```diff
@@ -13,7 +13,6 @@
         ]
     },
     install_requires=[
-        "colour",
```

Comment:
This has nothing to do with the current PR. Next time, please make sure that you PR files that are only relevant to the purpose of this PR.

Reply:
OK. I will make it. Or shall I close it and make a new one?